### PR TITLE
Setup command line interface

### PR DESCRIPTION
**Pull request by shana0440**
This PR setup the command line interface, the feature of the command will be the follow up PR.
- add build command
  - also get the files under the specify directory, so the follow PR only need to handle the compile part
- add compile command

### CODE REVIEW

File: build.rkt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#lang racket
+(provide build)
+
+(define (build dir-path)
+  (define files (filter
+    (lambda file
+      (string-suffix? (path->string (first file)) ".ss"))
+    (directory-list dir-path)))
```
Comment (dannypsnl) [inline build.rkt:8]: I guess you would like `fold-files` for the case

File: command.rkt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      (command-line
+        #:argv rest
+        #:args (dir-path)
+        (cons "build" dir-path)
+      )]
+    ["compile"
+      (command-line
+        #:argv rest
+        #:args (file-path)
+        (cons "compile" file-path))])))
```
Comment (dannypsnl) [inline command.rkt:21]: I guess later this would be in `main.rkt`?

File: compile.rkt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#lang racket
+(provide compile)
+
+(define (compile file-path)
+  (println file-path))
```
Comment (dannypsnl) [inline compile.rkt:5]: I guess this exactly wants the `compile-all` function in `main.rkt`

Comment (dannypsnl) [reply compile.rkt:5]: and the eof

**Review by dannypsnl — CHANGES_REQUESTED** [review]
Great start! I would need a list of what gets done in the PR, corresponding to #6, and you can click the checkbox directly when you feel you have completed a task :).

By the way, you can try [magic-racket](https://marketplace.visualstudio.com/items?itemName=evzen-wybitul.magic-racket) and [racket-langserver](https://github.com/jeapostrophe/racket-langserver) as development tool.

**Comment by shana0440** [comment]
@dannypsnl updated, thanks.
Rename `command.rkt` to `main.rkt` and handle the compile will be the follow up PR, it likely is a huge work for me so want to separate to different PR.

**Comment by dannypsnl** [comment]
> @dannypsnl updated, thanks.
> Rename `command.rkt` to `main.rkt` and handle the compile will be the follow up PR, would like to make PR small as possible since I'm not familiar with racket.

Get it

**Pull request merged**